docs/source/selection.rst

```diff
@@ -0,0 +1,76 @@
+Data Sub-Selection
```

Comment:
This file is not included in the main doctree

Reply:
Done